src/styles/blocks/page.scss

```diff
@@ -0,0 +1,50 @@
+html {
+  font-family: "Cinzel", serif;
+  font-weight: 700;
+}
+
+.page {
+  box-sizing: border-box;
+  scroll-behavior: smooth;
+
+  &__body {
+    margin: 0;
+
+    &--with-menu {
+      overflow: hidden;
+    }
+  }
+
+  &__menu {
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+
+    transform: translateX(-100%);
+    transition: transform 0.5s linear;
```

Comment:
better to have variable for transition time. Because it is the same everywhere.

Reply:
fixed